pages/mentions_legales.php

```diff
@@ -0,0 +1,44 @@
+<h1>Mentions Légales</h1>
```

Comment:
no doctype in this document ? no header? no footer ?

Reply:
deleted, this file was a modal